server/githubsrm/administrator/models.py

```diff
@@ -75,12 +96,21 @@ def verify_admin(self, email: str, password: str) -> bool:
         Returns:
             bool
         """
-        hash_password = sha256(password.encode()).hexdigest()
         if value := self.db.admins.find_one(
-            {"$and": [{"password": hash_password}, {"email": email}]}
+                {"email": email}
```

Comment:
We are using custom errors here. Try to change the function to fit the errors. You should find it in the main branch.

Reply:
What do you mean by changing the function?? Im a little confused

Reply:
![image](https://user-images.githubusercontent.com/63897835/133379903-1ec12cda-51cc-4192-b2fa-005977e774cd.png)
Your logic for the function is correct. Just incoporate the custom errors. This was the function previously for your reference. Hope this answers your question.

Reply:
Alright thanks just check out what i've done :) Thanks again